.azure/pipelines/ci.yml

```diff
@@ -22,515 +22,579 @@ variables:
   value: true
 - name: _TeamName
   value:  AspNetCore
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - name: _BuildArgs
-    value: /p:TeamName=$(_TeamName)
-      /p:OfficialBuildId=$(Build.BuildNumber)
-- ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+- name: _DotNetPublishToBlobFeed
+  value: true
+- name: _PublishUsingPipelines
+  value: true
+- name: _DotNetArtifactsCategory
+  value: .NETCORE
+- name: _DotNetValidationArtifactsCategory
+  value: .NETCORE
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
   - name: _BuildArgs
     value: ''
+  - name: _PublishArgs
+    value: ''
+  - name: _SignType
+    value: ''
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
When do we make PRs to internal? Security fixes?

Reply:
Yup